layers/ip6.go

```diff
@@ -34,6 +75,7 @@ type IPv6 struct {
 	HopLimit     uint8
 	SrcIP        net.IP
 	DstIP        net.IP
+	RoutingDstIP net.IP
```

Comment:
Which could be DstIP if there is no Routing header.

On Fri, Sep 11, 2015 at 10:22 AM, Graeme Connell <gconnell@google.com>
wrote:

> Yeah, it'd return the "next hop".
>
> On Fri, Sep 11, 2015 at 10:04 AM, randstr <notifications@github.com>
> wrote:
>
>> In layers/ip6.go
>> <https://github.com/google/gopacket/pull/120#discussion_r39292520>:
>>
>> > @@ -34,6 +75,7 @@ type IPv6 struct {
>> >  	HopLimit     uint8
>> >  	SrcIP        net.IP
>> >  	DstIP        net.IP
>> > +	RoutingDstIP net.IP
>>
>> If I understand correctly RoutingDst() will look ahead interpreting the
>> IPv6 layer payload bytes until and if we find an IPv6 routing extension
>> header?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/google/gopacket/pull/120/files#r39292520>.
>>
>
>